community/dbms/src/test/java/org/neo4j/commandline/dbms/ImportCommandTest.java

```diff
@@ -198,67 +198,85 @@ public void shouldPrintNiceHelp() throws Throwable
+                            "  --id-type=<STRING|INTEGER|ACTUAL>\n" +
+                            "      Each node must provide a unique id. This is used to find the correct nodes\n" +
+                            "      when creating relationships. Possible values are:\n" +
+                            "        STRING: arbitrary strings for identifying nodes,\n" +
+                            "        INTEGER: arbitrary integer values for identifying nodes,\n" +
+                            "        ACTUAL: (advanced) actual node ids.\n" +
+                            "      For more information on id handling, please see the Neo4j Manual:\n" +
+                            "      https://neo4j.com/docs/operations-manual/current/tools/import/\n" +
+                            "      [default:STRING]\n" +
+                            "  --input-encoding=<character-set>\n" +
+                            "      Character set that input data is encoded in. [default:UTF-8]\n" +
+                            "  --ignore-extra-columns=<true|false>\n" +
+                            "      If un-specified columns should be ignored during the import.\n" +
+                            "      [default:false]\n" +
+                            "  --ignore-duplicate-nodes=<true|false>\n" +
+                            "      If duplicate nodes should be ignored during the import. [default:false]\n" +
+                            "  --ignore-missing-nodes=<true|false>\n" +
+                            "      If relationships referring to missing nodes should be ignored during the\n" +
+                            "      import. [default:false]\n" +
+                            "  --multiline-fields=<true|false>\n" +
+                            "      Whether or not fields from input source can span multiple lines, i.e.\n" +
+                            "      contain newline characters. [default:false]\n" +
+                            "  --delimiter=<,|TAB|;|||:|#>\n" +
+                            "      Delimiter character between values in CSV data. [default:,]\n" +
+                            "  --array-delimiter=<,|TAB|;|||:|#>\n" +
+                            "      Delimiter character between array elements within a value in CSV data.\n" +
+                            "      [default:;]\n" +
+                            "  --max-memory=<max-memory-that-importer-can-use>\n" +
+                            "      Maximum memory that importer can use for various data structures and\n" +
+                            "      caching to improve performance. If left as unspecified (null) it is set to\n" +
+                            "      90% of (free memory on machine - max JVM memory). Values can be plain\n" +
+                            "      numbers, like 10000000 or e.g. 20G for 20 gigabyte, or even e.g. 70%.\n" +
+                            "      [default:90]\n",
```

Comment:
Generally the theme of copying these descriptions into three locations isn't great is it?